### ISSUE TITLE
Split training programs and events

### ISSUE THREAD

**Issue by plexish** [StampyAI/AISafety.com#43]

**Comment by melissasamworth** [comment]
@plexish I think this makes sense. Can you tell me more about what training is and I guess what events are too?

**Comment by plexish** [comment]
training programs are generally 3-4+ days, often several weeks long. events are as short as a single talk, often a weekend, sometimes as long as a week.

there is a bit of overlap in length, but you can usually tell whether something is a training program by how structured and class-like it is, vs how talk/unconference/retreat-like it is

**Comment by ricewithabee** [comment]
I've given some thought to the whole Courses vs Events vs Training programs thing and come to the conclusion that I also think it would be best having the three separated out. This would entail adding a page for "Training programs".

Seems like we agree on this. If so, @melissasamworth I'm happy to have a whack at creating it?

**Comment by melissasamworth** [comment]
Right, I've been deliberating over separating out courses vs. training and can see how the time aspect might make that worth differentiating. I already designed most of it right before I read your comment @ricewithabee but I would be happy to let you try designing in the future. It might be worth me leading you through my process on this!

**Comment by ricewithabee** [comment]
Update: this is something Melissa has to do so top of backburner for now.
